packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -112,13 +133,25 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
     >
       {actionListHeaderChild}
       <StyledListBoxWrapper
+        sections={sectionData}
+        windowSize={5}
+        initialNumToRender={5}
+        bouncesZoom={false}
+        bounces={false}
```

Comment:
Lets keep bounces as true? That's the expected behaviour on iOS

Reply:
Bounces messes up with the Section list headers in iOS

https://user-images.githubusercontent.com/35374649/231438994-2456b83e-c1f4-4628-b903-00107b37c3e2.mov

Reply:
Okay added bg white looks okay. There was still a bit of a gap even after adding bg color because of padding in the container, had to use a bit of negative margin hacks to fix it.